interface/ceed-elemrestriction.c

```diff
@@ -1490,10 +1490,10 @@ int CeedElemRestrictionGetElementSize(CeedElemRestriction rstr, CeedInt *elem_si
 
 /**
 
-  @brief Get the number of points in the l-vector for a points `CeedElemRestriction`
+  @brief Get the number of points in the offsets array for a points `CeedElemRestriction`
```

Comment:
Maybe 'number of points being restricted to' or 'number of points is the E-vector' ?

Does the constructor need similar updated language?

Reply:
the constructor uses the "offsets array" language

Reply:
fair enough